datajunction-server/datajunction_server/api/namespaces.py

```diff
@@ -61,23 +66,25 @@ def create_node_namespace(
 
 @router.get(
     "/namespaces/",
-    response_model=List[str],
+    response_model=List[NamespaceOutput],
     status_code=200,
 )
 def list_namespaces(
     session: Session = Depends(get_session),
-) -> List[str]:
+) -> List[NamespaceOutput]:
```

Comment:
Will this cause the Python client to fail? I think at the moment it assumes that this endpoint returns a list of strings

Reply:
Yep, fixed! I just parse out the names right after the call.

Reply:
In the future we might want to use the full payload to generate an array of objects or maybe even just a single NodeList object that prints as a nice looking table. 🤔